src/coreclr/vm/codeman.cpp

```diff
@@ -1184,10 +1184,12 @@ void EEJitManager::SetCpuInfo()
     {
 #if defined(TARGET_X86) || defined(TARGET_AMD64)
         EEPOLICY_HANDLE_FATAL_ERROR_WITH_MESSAGE(COR_E_EXECUTIONENGINE, W("\nThe current CPU is missing one or more of the following instruction sets: SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2, POPCNT\n"));
-#elif defined(TARGET_APPLE_MOBILE_SIMULATOR)
-        // iOS and tvOS simulators don't have required instruction sets
 #elif defined(TARGET_ARM64) && (defined(TARGET_WINDOWS) || defined(TARGET_APPLE))
+#if defined(FEATURE_INTERPRETER)
+        // Hardware intrinsics are disabled in CoreCLR interpreter: https://github.com/dotnet/runtime/issues/117948
+#else
```

Comment:
This looks to be the same issue as https://github.com/dotnet/runtime/issues/119008

and rather we need to update https://github.com/dotnet/runtime/blob/28216aca3f72a4fb22fd6d4c6714be039873e726/src/native/minipal/cpufeatures.c#L505-L514 to only require LSE for `OSX/MacCatalyst`

This and the NAOT path then need to also do the same, so the iOS/tvOS paths go down the Linux path where it only checks for `AdvSimd`

Reply:
I think there is a different namespace used for devices/simulators. I confirmed that this check works locally, and it will likely fix the mentioned issue:
```
    if ((((sysctlbyname("hw.optional.AdvSIMD", &valueFromSysctl, &sz, NULL, 0) != 0) || (valueFromSysctl == 0)) &&
         ((sysctlbyname("hw.optional.arm.AdvSIMD", &valueFromSysctl, &sz, NULL, 0) != 0) || (valueFromSysctl == 0))) ||
        (sysctlbyname("hw.optional.arm.FEAT_LSE", &valueFromSysctl, &sz, NULL, 0) != 0) || (valueFromSysctl == 0))
    {
```

Reply:
Gotcha, thanks for investigating/confirming!

This doesn't look to be covered by the current Apple docs: https://developer.apple.com/documentation/kernel/1387446-sysctlbyname/determining_instruction_set_characteristics/